internals/models.py

```diff
@@ -1740,6 +1740,36 @@ def remove_as_component_owner(self, component_name):
         component_name, remove_as_owner=True)
 
 
+class APIOwner(DictModel):
```

Comment:
It would be good to generalize this a bit to other OWNERS files as well.  E.g., name it something like `OwnersFile`.